packages/dendron-cli/src/commands/launchEngineServer.ts

```diff
@@ -101,9 +106,47 @@ export class LaunchEngineServerCommand extends CLICommand<
       vaults,
       ws: wsRoot,
     });
+
     if (init) {
       this.L.info({ ctx, msg: "pre:engine.init" });
       const out = await engine.init();
+
+      // These events will only upload if the upload action completes before the
+      // CLI command completes. They are uploaded on a best effort basis.
+      engine.onEngineNoteStateChanged((entries) => {
+        const createCount = extractNoteChangeEntriesByType(
+          entries,
+          "create"
+        ).length;
+
+        if (createCount > 0) {
```

Comment:
Why not consolidate these into a single event with properties for what kind of change it is? I'm asking since one of the Amplitude review items was that we had a lot of events repeated after one another.

Reply:
At first I was thinking that it'd be easier to build analytical views with them as separate events, but on second thought it should be fine as a consolidated event. These will likely be noisy, so I can make the adjustment.  👍🏼